### DISCUSSION TITLE
3.4.0

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#26] [Announcements]
Blender 3.4.0, commit [175bd3820151](https://developer.blender.org/diffusion/B/history/v34/;175bd3820151).
Build environment:

    Visual Studio 2019, version 16.11.21
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/3.4.0'>3.4.0</a>.</em>

**Comment by lnesuper** [comment]
whith time creat 3.3.2lts version?

**Reply by nalexandru** [reply]
Done #27.

**Reply by lnesuper** [reply]
thy

**Comment by patientAtGithub** [comment]
Thank you VERY MUCH again!

**Comment by waltage80** [comment]
Eevee material crush the Blender
[bug.crash.txt](https://github.com/nalexandru/BlenderCompat/files/10252507/bug.crash.txt)

**Reply by nalexandru** [reply]
Please open an issue and provide the steps to reproduce the crash. In addition to the file attached here, start Blender using 'blender_debug_gpu.cmd', repeat the steps that cause the crash and upload the two files in the directory that will appear after Blender crashes.

**Reply by waltage80** [reply]
Here is crash after simple material on cube
[blender_system_info.txt](https://github.com/nalexandru/BlenderCompat/files/10252945/blender_system_info.txt)
[blender_debug_output.txt](https://github.com/nalexandru/BlenderCompat/files/10252946/blender_debug_output.txt)

PS Cycles Render work fine
PPS Nvidia driver -Latest

**Reply by patientAtGithub** [reply]
Hi,
I just tried it on my very old laptop with an NVidia GTX 460M (2011) and I am not getting any crashes in Eevee. I applied a single material and multiple materials on a single object.

**Reply by waltage80** [reply]
so this stupid logs don't show the reason of crush?

**Reply by patientAtGithub** [reply]
I am just a user. I am not **nalexandru**. I don't know what the logs show. I just wanted to say that Eevee doesn't crash here with this simple procedure, in case it is useful. Let's see what **nalexandru** can tell from the logs.

**Reply by patientAtGithub** [reply]
Hey waltage80,
If you run Blender 3.4 without any extra addons and with the default preferences does it still crash? I only have very few addons enabled and they are all from those that come with Blender. Your GPU is much newer than mine. 
(note: I am not nalexandru)

**Reply by waltage80** [reply]
Now run Blender 3.4 without any extra addons still crash
Total default Blender don't crush
But i have now so much preferences to do((
Is it some preferences wich can spoil the Blender Eevee?

**Reply by nalexandru** [reply]
I have opened an issue for this, please see #29.

**Comment by 446367190** [comment]
Can't download

**Reply by nalexandru** [reply]
Can you provide more details ? What is the problem exactly ?
Downloads are handled by Github, there was a problem some time ago that cauzed a "Failed - Needs authorization", which i was unable to reproduce.

The only thing that comes to mind is to try downloading with a different browser.

**Reply by 446367190** [reply]
The download process will be interrupted，You need to retry several times to download。I have finished downloading. Thank you very much。

**Comment by lnesuper** [comment]
when to update 3.4.1version?
